src/data_sets.h

```diff
@@ -4045,17 +4045,52 @@ enum eGpxStatus
     GPX_STATUS_FAULT_SYS_CRITICAL               = (int)0x80000000,
 };
 
+typedef enum {
+    kReset = 0,
+    kInit,
+    kRun,
+    kPassthrough,
+    kFwInit,    // initializing into FwUpdate mode (prep for code injections)
+    kFwUpdate,  // ready and able to accept code injections
+    kError,
+    kShutdown,
```

Comment:
Because data_sets.h gets included into our customers projects, these short names are prone to cause conflicts with other similar variable names.  I think we should stick with our convention used with other enums defined in data_sets.h and name these something more like;

```
typedef enum {
    GPX_RUN_STATE_RESET = 0,
    GPX_RUN_STATE_INIT,
    GPX_RUN_STATE_RUN,
    GPX_RUN_STATE_PASSTHROUGH,
    GPX_RUN_STATE_FWINIT,    // initializing into FwUpdate mode (prep for code injections)
    GPX_RUN_STATE_FWUPDATE,  // ready and able to accept code injections
    GPX_RUN_STATE_ERROR,
    GPX_RUN_STATE_SHUTDOWN,
} GPXGnssRunState_t;
```

Reply:
done